_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: post
+title:  "Introduction to system tests"
+date: 2020-09-07
+author: maros_orsak
+---
+
+Lessons learn from past years shown that testing is the essential part of the software cycle.
+We have known many failures that happened because of neglect testing.
+In Strimzi, we exactly know how we should test the features.
+Follow me on this blog post and see the basic view of our tests.
+
+In this blog post we are gonna look at system tests and I will give you motivation why you should look at it.
+The content is as follows:
+
+# Content
```

Comment:
I think people have commented about _content_. Not sure it's necessary considering the size of the post

Reply:
Ok, I will remove it.